plugins/reporters/evaluated-model/src/main/kotlin/EvaluatedModelMapper.kt

```diff
@@ -169,21 +173,26 @@ internal class EvaluatedModelMapper(private val input: ReporterInput) {
 
         input.ortResult.getProjects().forEach { project ->
             val pathExcludes = input.ortResult.getExcludes().findPathExcludes(project, input.ortResult)
+            val pathIncludes = input.ortResult.getIncludes().findPathIncludes(project, input.ortResult)
             val dependencies = input.ortResult.dependencyNavigator.projectDependencies(project)
-            if (pathExcludes.isEmpty()) {
+            if (pathExcludes.isEmpty() && pathIncludes.isEmpty()) {
```

Comment:
I think there is a problem here: `pathIncludes.isEmpty()` can mean two things:
* There are no path includes defined at all so this project is included.
* There are path includes but none match this project so it is excluded.

I wonder if for the case that no path includes are defined at all, `Includes` should return a default path include which matches `**` to make the usage unambiguous.

Reply:
Am I right that semantics is as follows: A file is included, if it is not matched by any exclude `OR` if it is matched by an include?